django_freeradius/tests/test_api.py

```diff
@@ -16,3 +19,133 @@ def test_users_unauthorize(self):
                                     {'username': 'baldo', 'password': 'ugo'})
         self.assertEqual(response.status_code, 401)
         self.assertEqual(response.data, {'control:Auth-Type': 'Reject'})
+
+    def test_accounting_start(self):
+        RadiusAccounting.objects.create(session_id='35000006', unique_id='75058e505ac30c5b4b9c686ee853b70f',
```

Comment:
why are we creating an accounting through the ORM and not directly through the API?
I think it's more correct to do this operation directly through the post request